vmaas_sync/advisory_sync.go

```diff
@@ -252,3 +264,21 @@ func vmaasErrataRequest(iPage int, modifiedSince *string) (*vmaas.ErrataResponse
 	vmaasCallCnt.WithLabelValues("success").Inc()
 	return vmaasDataPtr.(*vmaas.ErrataResponse), nil
 }
+
+func checkNumberOfAdvisories() error {
+	var databaseAdvisoriesCount int64
+	database.Db.Table("advisory_metadata").Count(&databaseAdvisoriesCount)
+	errataResponse, err := vmaasErrataRequest(0, nil, true)
+	if err != nil {
+		return errors.Wrap(err, "Advisories check failed on vmaas request")
+	}
+
+	errataCount := int64(errataResponse.GetPages()) + 1
+	if databaseAdvisoriesCount != errataCount {
+		err = syncAdvisories(time.Now(), nil)
+		if err != nil {
+			return errors.Wrap(err, "Full advisories sync failed")
+		}
+	}
+	return nil
+}
```

Comment:
Also please update commit message to: `SPM-987: Add check after advisories incremental sync` as we will also need to use it for packages sync and it's not impelmented now.